pkg/provider/validators.go

```diff
@@ -28,6 +31,13 @@ func validateProjectId(v interface{}, path cty.Path) diag.Diagnostics {
 	return diags
 }
 
+func ProjectIdValidator() validator.String {
```

Comment:
It would be nice to check if we can add this validator to the other parts of the code. It would be great to have a GitHub Issue for it, and then we can add this validator to other parts of the code in a separate PR. 

This function would stay here in this PR. Up to you.

Reply:
Thanks! I prefer to keep this function in this PR